hr_360_view/wizard/hr_employee_search_wizard.py

```diff
@@ -270,13 +276,13 @@ def do_bankid(self):
                     self.bank_id_text = _("Error in communication with BankID")
         except KeyError:
             self.bank_id_text = _("Error in communication with BankID")
-            self.bank_id_ok = False			
+            self.bank_id_ok = False
         if orderRef:
             deadline = time.monotonic() + TIMEOUT
             time.sleep(9) # Give user time to react before polling.
             while deadline > time.monotonic():
                 res = bankid.service.verifieraIdentifiering(orderRef,'crm')
-                _logger.warn("res: %s" % res)
+                _logger.warning("res: %s" % res)
```

Comment:
Using warning in a loop is bad cause its have performance impact.
As Warning is a high log level its hard to mask without masking most other logging as well.
In loops its recommended to use INFO or DEBUG

Reply:
Maybe add a short sleep in this loop so to not use excessive polling.
Half a second maybe?